TUMFARSynchrony/experimental-hub
# eslint prettier different os

Currently, prettier is not working with Windows since Windows has different end of line endings. (CRLF vs LF) Adding the endOfLine: auto option, this should be fixed.
Furthermore, eslint is now enabled to look in all parent directories which might cause an error that some packages are missing (e.g. eslint-plugin-react). This can be fixed by adding the root option and setting it to true. 
Eslint describes this options as follows:
_By default, ESLint looks for configuration files in all parent folders up to the root directory. This can be useful if you want all of your projects to follow a certain convention, but can sometimes lead to unexpected results. To limit ESLint to a specific project, place "root": true inside the .eslintrc.* file or eslintConfig field of the package.json file or in the .eslintrc.* file at your project’s root level. ESLint stops looking in parent folders once it finds a configuration with "root": true._

Testing:
- prettier:
  - add a space at one file, stage it, make a commit, look if commit is working
- eslint:
  - build frontend
  - start frontend
  - if there are no errors thrown, then the changes should be working
